extensions/positron-python/package.json

```diff
@@ -1918,6 +1926,158 @@
                     }
                 },
                 "tags": ["positron-assistant"]
+            },
+            {
+                "name": "get_python_environment_details",
```

Comment:
Still have to test how these new tools interact with Assistant

Reply:
IIUC since these tools aren't tagged with `positron-assistant`, they will become available only in Agent mode in the Chat pane, where we already know we don't do a great job of tool selection (#8113).

I think it's fine to merge this in as is, and to do any further work here as part of #8113. Also because Assistant is still in "preview" and disabled by default.

Reply:
After some testing, you're right that it doesn't use these tools in Ask/Edit modes. In Agent mode it will use them, with undesirable results (here I already had a session running but it wants me to pick another, note it doesn't open our picker, etc.)
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/fb082ca2-585a-440b-b74c-48f3b76dd637" />